doc/dev/index.rst

```diff
@@ -502,60 +506,99 @@ from 20 minutes to three hours to complete, but it's worth the wait.
 When you fix a bug, it's a good idea to add a test. See the `Writing make
 check tests`_ chapter.
 
-Integration tests intro
------------------------
+Further sections
+----------------
+
+* Principles of make check tests
+* Where to find test results
+* How to interpret test results
+* Find the corresponding source code
+* Writing make check tests
+* Make check caveats
+
+Testing - integration tests
+===========================
 
 When a test requires multiple machines, root access or lasts for a
 longer time (for example, to simulate a realistic Ceph deployment), it
-is deemed to be an integration test. Integration tests are defined
-in the `ceph-qa-suite repository`_ and run with the `teuthology
-framework`_.
-
-A number of integration tests are run on a regular basis against the
-official Ceph repositories (on the master development branch and the
-stable branches). Traditionally, these tests are called "the nightlies"
-because most of the Ceph developers used to live and work in California and
-from their perspective the tests were run overnight. 
+is deemed to be an integration test. Integration tests are organized into
+"suites", which are defined in the `ceph-qa-suite repository`_ and run with
+the `teuthology-suite`_ command (in bare-metal testing environments such as
+the `Sepia lab`_) or its wrapper `teuthology-openstack`_ (in OpenStack
+tenants).
+
+A number of integration tests are run on a regular basis in the `Sepia
+lab`_ against the official Ceph repositories (on the master development
+branch and the stable branches). Traditionally, these tests are called "the
+nightlies" because the Ceph core developers used to live and work in
+the same time zone and from their perspective the tests were run overnight. 
 
 The results of the nightlies are visible at either http://pulpito.ceph.com/
 and http://pulpito.ovh.sepia.ceph.com:8081/ (depending on how the tests
-were run) and are also reported on the `ceph-qa mailing list
+were run, i.e. via `teuthology-suite`_ or `teuthology-openstack`_), and are
```

Comment:
I would stick to teuthology-suite for simplicity. I think teuthology-openstack will eventually be hidden from the user by ceph-workbench ceph-qa-suite eventually.